### ISSUE TITLE
[Tracking] - V1 Release of the Prysm Web UI

### ISSUE THREAD

**Issue by rauljordan** [prysmaticlabs/prysm-web-ui#116]
## Background

This is a tracking issue for all required items needed from the Angular side to promote the Prysm web UI from beta to v1. There are a few backend related issues that need to be resolved in parallel, but there are indeed low hanging fruit that would drastically improve the experience for users of the interface.

## Serious Bugs/Problems

- [x] When importing new validators into a current wallet, current balance is either "0" or "n/a" in web portal until beacon chain deposit confirmation #114
- [x] Ensure backend responses which are not a result of user interaction have sensible default instead of spamming internal server error at the bottom #100
- [x] We tell the user to drag and drop their `validator_keys` directory from the eth2 deposit cli, but the backend will fail because it includes the deposit_data.json file which is not a valid keystore. We can perhaps filter this deposit data json file out in the frontend. Also, the drag and drop does not allow dropping in full directories, so our text is misleading #110
- [x] Using a weak entropy password will only show an HTTP 400 bad request error. Meaning, a user created a password which meets the password requirements from the frontend, such as `aaaaaaaaaaaaaaaA$1` but is considered weak by the backend. The error is confusing and gives little context to the user. We need a way to force higher entropy passwords in the UI or better feedback to the user if this error occurs
- [x] Error messages just show the HTTP error code such as "500 internal server error" but not the actual message contents of the error, such as "beacon node is syncing and not ready to serve requests". This data is available in error.message, I believe
- [x] Adjust the zoom level of the peers map to not show the location as accurate as street address (user concern)
- [ ] If you run with --wallet-dir, web UI cannot find the wallet.  If you try to import the wallet from that datadir, you get a 500 error during the process.  Instead, if you just use the default --data-dir location, everything works (this is on Windows).

## Improvements

- [ ] Show Prysm version for beacon chain and validator in the web UI (preferably somewhere always visible such as header or sidebar) #107 
- [ ] If a user has > 5 keys pending activation or exit, these keys in the main dashboard are cut off and only 5 entries are displayed instead of some marker that > 5 exist #95
- [x] Consolidate all prysm backends into one to prevent cors, hosts, port-forwarding issues and complex configurations for users https://github.com/prysmaticlabs/prysm/issues/7984
- [x] Redirect the user to login page on 401 authorized issues (see the ErrorInterceptor ts file in core/, seems like it is not working at the moment or a route redirect needs to be added) #109

**Comment by rauljordan** [comment]
Closing in favor of https://github.com/prysmaticlabs/prysm-web-ui/issues/137
